exec/java-exec/src/main/java/org/apache/drill/exec/planner/logical/DrillOptiq.java

```diff
@@ -302,9 +302,12 @@ private LogicalExpression getDrillCastFunctionFromOptiq(RexCall call){
         castType = Types.required(MinorType.VARCHAR).toBuilder().setPrecision(call.getType().getPrecision()).build();
         break;
 
-      case "INTEGER": castType = Types.required(MinorType.INT); break;
-      case "FLOAT": castType = Types.required(MinorType.FLOAT4); break;
-      case "DOUBLE": castType = Types.required(MinorType.FLOAT8); break;
+      case "INTEGER": castType = Types.required(MinorType.INT);
```

Comment:
Could you please move the assigning to the next line here and in this class below

Reply:
done